build_centos.dockerfile

```diff
@@ -0,0 +1,46 @@
+# https://github.com/ghdl/ghdl/issues/1484
+# https://github.com/ghdl/ghdl/issues/1118
+
+#
+# Centos 7
+#
+
+FROM centos:centos7
+
+RUN yum update -y && yum install -y git curl bzip2 wget flex fontconfig libX11 zlib-devel
```

Comment:
Please, use a row for each package. It makes comparison and source code management easier.